cicecore/cicedyn/infrastructure/io/io_netcdf/ice_history_write.F90

```diff
@@ -742,6 +648,12 @@ subroutine ice_write_hist (ns)
          call ice_check_nc(status, subname// ' ERROR: global attribute io_flavor', &
                            file=__FILE__, line=__LINE__)
 
+         if (history_format == 'hdf5') then
+            status = nf90_put_att(ncid,nf90_global,'deflate',history_deflate)
+            call ice_check_nc(status, subname//' ERROR: defining attribte deflate', &
+                              file=__FILE__,line=__LINE__)
+         endif
+
```

Comment:
@apcraig I am not sure about adding this as an attribute. It makes is seem like its a global feature, when its actually at the variable level. You can already see the deflate level through ncdump (and probably other tools) without add it as an attribute.

Reply:
I'm happy to remove this again, it's not a critical thing.  Will do that now.